### PR TITLE
GRID-402: Default value for datetime field

### PR DESCRIPTION
**Pull request by rvalim**
The "defaultValue" is defined on "AdvancedFormat", OS column's parameters, to inject code in the screen. This property needs to be removed from "gridOptions" before initializing the grid.

Now we have two options for defaultValue:
 - DefaultValue column's parameter: a fixed value, unless you refresh the browser
 - AdvancedFormats - DefaultValue: a javascript function that can update values without refresh. Sample: "defaultValue: function() { return new Date(); }"

Screen to test (code not publish): https://pp-datagridteam-dev.outsystemsenterprise.com/Test_DataGridSample/Test_Grid_402.aspx

### CODE REVIEW

File: GridFramework.js [hunk 1]
```diff
@@ -885,6 +885,20 @@ GridOS.ComponentUtils = {
         }
     },
     
+    // **************************************************
+    // Initialize the grid
+    // GridOS.ComponentUtils.initGrid
```
Comment (ruipmendes) [inline GridFramework.js:890]: These nedd update

Comment (rvalim) [reply GridFramework.js]: It was missing a push =D

File: GridFramework.js [hunk 2]
```diff
@@ -885,6 +885,20 @@ GridOS.ComponentUtils = {
         }
     },
     
+    // **************************************************
+    // Responsable for cleaning columns for wijmo components
```
Comment (ruipmendes) [inline GridFramework.js:889]: Responsible

Comment (rvalim) [reply GridFramework.js]: done

**Pull request merged**